docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -57,25 +57,115 @@ For static onboarding, make sure that the following parameters are correctly spe
 
 For static onboarding, be sure to use the [Gateway static definition example](#gateway-static-definition-example) presented at the end of this article.
 
-## Establishing a trust relationship between domain Gateways and the Cloud Gateway
-
-The following keytool commands are examples of establishing a trust relationship between domain Gateways and the Cloud Gateway.
-
-- Import the public key certificate of all domain Gateways into the truststore of the Cloud Gateway.
-
-  Use the following example of the keytool command when the domain Gateways running on CA11 and CA32 import the certificate into the Cloud Gateway running on CA31:
-
-  `keytool -import -file keystore/ca11/local_ca/local_ca.cer -alias gateway_ca11 -keystore keystore/ca31/localhost/localhost.truststore.p12`
-
-  `keytool -import -file keystore/ca32/local_ca/local_ca.cer -alias gateway_ca32 -keystore keystore/ca31/localhost/localhost.truststore.p12`
-
-- Import a public key certificate of the Cloud Gateway into the truststore of all domain Gateways.
-
-  Use the following example of the keytool command  when the certificate of the Cloud Gateway running on CA31 is imported into the truststore of the domain Gateways running on CA11 and CA32:
-
-  `keytool -import -file keystore/ca31/local_ca/local_ca.cer -alias gateway_ca31 -keystore keystore/ca11/localhost/localhost.truststore.p12`
-
-  `keytool -import -file keystore/ca31/local_ca/local_ca.cer -alias gateway_ca31 -keystore keystore/ca32/localhost/localhost.truststore.p12`
+## Establishing a trust relationship between Domain API ML and the Central API ML
+
+For routing to work in a MultiTenancy configuration, the Central API ML must trust the Domain API ML, and vice versa for registration.
+In other words it is necessary that the root and, if applicable, intermediate public certificates be shared between Central API ML and Domain API ML. 
+
+The following example describes this relationship. 
+
+![Trust relation diagram](./diagrams/mt-trust-relations.png)
+
+On this example diagram:
+
+Central APIML is installed on system X, Domain APIMLs are installed on system Y and Z.
+For secure communication Domain APIML 1 and 2 are using different private keys signed with different public keys (they don't trust each other).
+Central APIML should have all public keys from the certificate chain of all Domain APIMLs (DigiCert Root CA, DigiCert Root CA1, DigiCert CA) added to its truststore to be able to register them, otherwise Central APIML will not trust them.
+Central APIML is using a private key which is sign by Local CA public key for secure communication. 
+Domain APIML 1 and 2 should have Local CA public key to be able to accept the routing requests from Central APIML, otherwise Domain APIMLs will not trust it.
+On this diagram you can see all added certificates surrounded by red dashed line.
```

Comment:
```suggestion
Central API ML is installed on system X, Domain API MLs are installed on systems Y and Z.

* To establish secure communications, "Domain API ML 1" and "Domain API ML 2" are using different private keys signed with different public keys (they don't trust each other).
* Central API ML should have all the public keys from the certificate chains of all Domain API MLs ("DigiCert Root CA", "DigiCert Root CA1", "DigiCert CA") added to its truststore to be able to register them, otherwise the Central API ML will not trust them.
* Central API ML is using a private key which is signed by "Local CA" public key for secure communication. 
"Domain API ML 1" and "Domain API ML 2" should have Local CA's public key to be able to accept the routing requests from Central API ML, otherwise Domain API MLs will not trust it.
On this diagram you can see all added certificates surrounded by red dashed line.
```